packages/program-boilerplate/package.json

```diff
@@ -17,10 +17,14 @@
     "@babel/cli": "^7.1.5",
```

Comment:
It doesn't look like we need or are using babel anymore, these deps could be removed